setup.py

```diff
@@ -36,5 +36,5 @@
         'Topic :: Internet :: WWW/HTTP',
     ],
     test_suite='tests',
-    tests_require=['mock']
+    tests_require=['mock', 'pytest']
```

Comment:
They're alternatives indeed. We "had" to add that for Debian packaging. shub may not need it, that's true